.pre-commit-config.yaml

```diff
@@ -13,7 +13,8 @@ repos:
     -   id: blacken-docs
         additional_dependencies: [black==20.8b1]
 - repo: https://github.com/pre-commit/mirrors-prettier
-  rev: "v3.0.0-alpha.0"
+  # The version "v3.0.0-alpha.0" causes the formatted javascript code to fail parsing by PhantomJS.
+  rev: "v2.7.1"
   hooks:
     -   id: prettier
         types_or: [css, javascript]
```

Comment:
How about this?

```suggestion
        types_or: [css, javascript]
        args: [--trailing-comma=none]
```

Reply:
Worked on #140

Reply:
Actually I will try to add to `,prettirrc`.